Skip repartitioning on nemesis test

roystgnr opened the pull request:
The gold standard was generated with a non-repartitioning libMesh, so
we should stick to that even if the libMesh behavior changes.

This is intended to fix the issue discovered via testing of
https://github.com/libMesh/libmesh/pull/625 and reported in #5563

roystgnr committed:
Skip repartitioning on nemesis test

The gold standard was generated with a non-repartitioning libMesh, so
we should stick to that even if the libMesh behavior changes.

This is intended to fix the issue discovered via testing of
https://github.com/libMesh/libmesh/pull/625 and reported in #5563

friedmud commented:
@roystgnr I think I would like a second test in that directory that DOES test repartitioning.

Would you like me to do it and it put in as a PR to you... or would you mind doing it?

roystgnr commented:
There's a chicken-and-egg problem here, isn't there?  If we generate the gold standard from https://github.com/libMesh/libmesh/pull/625 then it won't pass MooseBuild until after that's merged and the MOOSE libMesh is updated.

friedmud commented:
That's right.  That's the right way to do it.

Add a commit to your PR that has a test in it that doesn't pass.  It will fail MooseBuild.

Once repartitioning is in libMesh add another commit to your PR to update the libMesh submodule.  That will cause the tests to be rerun with the new libMesh and they will _pass_.

Then we merge!

roystgnr committed:
Skip initial repartitioning of Nemesis meshes

Otherwise by the time we get a "skip_partitioning" instruction from
the input file, it's too late.

Initial repartitioning is probably a waste of time, not just a killer
for repeatability.

roystgnr committed:
Add nemesis_repartitioning_test

In theory this should fail with the current MOOSE libMesh, then
succeed once we pull in the changes from
https://github.com/libMesh/libmesh/pull/625
